SmartDeviceLink/SDLMutableDataQueue.m

```diff
@@ -10,6 +10,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+NSInteger const MAXDATACOUNT = 200;
```

Comment:
The changes in this class have nothing to do with this issue. Please revert.

Reply:
@joeljfischer done in 8737234